docs/docs/getting-started-aws/index.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: doc
+title: getting-started-aws
+permalink: /docs/getting-started-aws
+parent: /docs/home
+---
+
+# Getting Started with AWS
+
+AWS is a full-featured Cloud provider that is also fairly easy to integrate with Koding:
+
+1. **[Sign Up](#sign-up)**
+2. **[Subscribe to EC2](#subscribe)**
+3. **[Get your Access Key ID & Secret Access Key](#get-key)**
+4. **[Create your Stack](#create-stack)**
+5. **[Set your Credentials](#set-creds)**
+6. **[Save your Stack and Initialize](#save-init)**
+
+***
+
+## Sign Up <a name="sign-up"></a>
+
+Like with all cloud providers, AWS requires you to sign up:
+
+1. Go to https://aws.amazon.com/free/.
+2. Click Sign In to the Console. This will start the process of signing up after you select "I am a new user."
+3. Go through adding your Contact Info, Payment Info, verifying your identity, choosing a support plan. 
+4. Confirm your information and you should be good to go!
+
+[Learn more about pricing](https://aws.amazon.com/ec2/pricing/) for AWS.
+
+***
+
+## Subscribe to EC2 <a name="subscribe"></a>
+
+Now that you've signed up for AWS, you must subscribe to the Elastic Compute Cloud (EC2):
+
+1. Log into the AWS Console [here.](https://console.aws.amazon.com/console/home)
+2. Open the Services menu from the top navbar. 
+3. In that menu, select EC2 from the Compute category.
+4. You'll be taken to a screen prompting you to subscribe.  
+
+> This requires verification and can take some time.
+
+***
+
+## Get your Access Key ID & Secret Access Key <a name="get-key"></a>
+
+In order for Koding to integrate with EC2, we'll need your Access Key ID and Secret Access Key. 
+
+> AWS provides a guide for getting your Access Key and ID: http://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html
+
+We've copied that guide here:
+
+1. Use your AWS account email address and password to sign in to the AWS Management Console.
+> If you previously signed in to the console with IAM user credentials, your browser might open your IAM user sign-in page. You can't use the user sign-in page to sign in with your root credentials. Instead, choose Sign in using AWS Account credentials near the bottom of the page to go to the account sign-in page. In the upper right of the console, choose the account name or number and then choose Security Credentials.
```

Comment:
The final sentence is a step on its own and should be separate from this paragraph.